src/game.rs

```diff
@@ -38,6 +41,15 @@ impl Deserialize for PlayerId {
     }
 }
 
+impl<'a> FromParam<'a> for PlayerId {
```

Comment:
Ah, this is neat, if I'm understanding it correctly. This allows Rocket to correctly type the parameter passed into the nose_goes function in `api.rs`. 

What happens if junk is passed in? The request gets dropped?